delivery_auto_refresh/models/sale_order.py

```diff
@@ -32,6 +32,7 @@ def _onchange_partner_id(self):
                         lambda x: x in order.available_carrier_ids._origin
                     )
                 )
+        return
```

Comment:
If this is about the precommit warning this is just a trick with no effect. The proper way is to assign the `super` call to a variable that you're going to return at a given point.